packages/core/injector/container-scanner.ts

```diff
@@ -35,7 +35,7 @@ export class ContainerScanner {
       : metatypeOrToken;
     const instanceWrapper = dependencies.get(name as string);
     if (!instanceWrapper) {
-      throw new UnknownElementException();
+      throw new UnknownElementException(name.toString());
```

Comment:
Should we add an extra check against `null` / `undefined` values here? `name && name.toString()`

Reply:
```suggestion
      throw new UnknownElementException(name && name.toString());
```

so @andreimc can accept it easily ;)